packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -56,6 +56,7 @@ export type GridConfig<T extends { __typename?: string }> = {
     delete?: boolean;
     copyPaste?: boolean;
     readOnly?: boolean;
+    exposeFilters?: boolean;
```

Comment:
And here `prefilterable`?

Reply:
or `filterProps`? (to make clear that is generates props for the generated component)